### ISSUE TITLE
0.11-M4 backward compatibility

### ISSUE THREAD

**Issue by shadius** [akka/reactive-kafka#186]
Hi,

I just have a technical question - reactive kafka 0.11-M4 supports kafka 0.10.0, I guess it also uses the same version of kafka client. Does it mean we can't use this version against our kafka server 0.9.0 and we have to follow official migration steps from 0.9 to 0.10?

Thanks for help

**Comment by jkyamog** [comment]
I had to do this just recently.  So I just decided to upgrade the broker/server to 0.10.0.  Since we had to services that used 0.11-M4 reactive-kafka, one in scala and the other in java.  You just have to make sure the parameters in server.properties inter.broker.protocol.version and log.message.format.version is set properly.  Looking at this it seems there are also some java api changes, so I don't think reactive-kafka 0.11 will support 0.9.x kafka.  see more details here: http://kafka.apache.org/documentation.html#upgrade
